lxd-migrate/utils.go

```diff
@@ -143,6 +122,32 @@ func transferRootfs(ctx context.Context, op lxd.Operation, rootfs string, rsyncA
 	return nil
 }
 
+func transferRootfsForConversion(ctx context.Context, op lxd.Operation, rootfs string, rsyncArgs string, instanceType api.InstanceType) error {
```

Comment:
rootfs in this context seems a misnomer as it could be either a filesystem or a block disk.

How about `transferRootDiskForConversion`?

Reply:
This seems better. However, I would also rename `transferRootfsForMigration` into `transferRootDiskForMigration` as both functions do the same (trnasfer either rootfs or block volume)

Reply:
sounds good